jobs/cloud_controller_ng/templates/bpm.yml.erb

```diff
@@ -12,6 +12,22 @@ def mount_nfs_volume!(config)
   end
 end
 
+def enable_fog_debugging!(config)
+    if p("cc.log_fog_requests")
+        case p("cc.packages.fog_connection.provider", "").downcase
+        when "azurerm"
+          config["env"]["FOG_DEBUG"] = true
+        when  "aliyun"
+          config["env"]["ALIYUN_OSS_SDK_LOG_LEVEL"] = "debug"
+          config["env"]["FOG_DEBUG"] = true
+          config["env"]["DEBUG"] = true
+        else
+          config["env"]["DEBUG"] = true
```

Comment:
Is `DEBUG` redundant with the addition of the `FOG_DEBUG` variable?  From looking at this [commit](https://github.com/fog/fog-core/commit/685ad2c8c4c43351281663079bc5911234bbc36e), it seems like `FOG_DEBUG` doesn't actually introduce new debug information?  Or does it depend on the specific blobstore provider gem?

Reply:
Eventually, DEBUG will be redundant and can be removed. But FOG_DEBUG was introduced in fog-core just recently and is not yet released as far as I can see. Therefore I would stick with setting both FOG_DEBUG and DEBUG with the exception of Azure.
On Azure, setting DEBUG leads to full request/response logging including sensitive information and response bodies so we can't use DEBUG. FOG_DEBUG is handled in fog-azure-rm (branch fog-arm-cf) explicitly.